src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageViewModel.cs

```diff
@@ -239,32 +271,40 @@ public bool HasLaunchOption
             }
         }
 
-        public bool IsExecutable
+        public bool SupportsExecutable
         {
-            get
-            {
-                if (SelectedLaunchType == null)
-                {
-                    return false;
-                }
+            get {  return ActiveProviderSupportsProperty(UIProfilePropertyName.Executable); }
```

Comment:
Regardless of the style you settle on, all of the properties in the class should have the same formatting. The ones below have everything on a newline.